src/compositions/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,38 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Type } from "@compositions/FieldsetHeader/FieldsetHeader";
+import { FormControl } from "@compositions/FormControl/FormControl";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
+import { TextInput } from "@elements/TextInput/TextInput";
+import { action } from "@storybook/addon-actions";
+import { Meta, Story } from "@storybook/react";
+import { Accordion as AccordionComponent, AccordionItem, AccordionProps } from "./Accordion";
+
+export default {
+    title: "Compositions/Accordion",
+    component: AccordionComponent,
+} as Meta<AccordionProps>;
+
+export const Accordion: Story<AccordionProps> = () => (
+    <AccordionComponent>
```

Comment:
```suggestion
import { Accordion, AccordionItem, AccordionProps } from "./Accordion";

export default {
    title: "Compositions/Accordion",
    component: Accordion,
} as Meta<AccordionProps>;

export const Default: Story<AccordionProps> = () => (
    <Accordion>
```

Reply:
this way it creates a subitem which is not actually useless. IMO as long as there is only one story i would go with the current code
![image](https://user-images.githubusercontent.com/2420769/128200931-12731965-f6e0-49d4-b5e3-530422510584.png)
![image](https://user-images.githubusercontent.com/2420769/128200813-03829163-c8a9-4599-afc2-1cd1b730cc09.png)